### PR TITLE
blurlock: add page, i3lock: refresh

### PR DESCRIPTION
**Pull request by marchersimon**
**Version of the command being documented:**
[blurlock](https://gitlab.manjaro.org/packages/community/i3/i3exit/-/blob/master/blurlock) at commit `24c8de0c1ba863fd86d5f9b185f4c27dacacdc4c`
i3lock: version 2.13 © 2010 Michael Stapelberg

### CODE REVIEW

File: pages/linux/blurlock.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# blurlock
+
+> A simple wrapper around the i3 screen locker `i3lock`, which blurs the screen.
+> See also `i3lock`.
```
Comment (navarroaxel) [inline pages/linux/blurlock.md:4]:
```suggestion
> See also: `i3lock`.
```

File: pages/linux/i3lock.md [hunk 2]
```diff
@@ -3,22 +3,30 @@
...
+- Lock the screen with a simple color background (rrggbb format):
 
-`i3lock -i {{path/to/picture.png}}`
+`i3lock --color {{0000ff}}`
 
-- Disable the unlock indicator (removes feedback on keypress):
+- Lock the screen to a PNG background:
 
-`i3lock -u`
+`i3lock --image {{path/to/picture.png}}`
```
Comment (navarroaxel) [inline pages/linux/i3lock.md:16]: Why not `file`? easier to translate.
```suggestion
`i3lock --image {{path/to/file.png}}`
```

File: pages/linux/blurlock.md (hunk 1 above)
Comment (marchersimon) [reply pages/linux/blurlock.md:4]: We use `See also ` 87 times and `See also:` 51 times. We should standardize that. Not sure to what, but I'll just apply your suggestion for now.

File: pages/linux/i3lock.md [hunk 3]
```diff
@@ -3,22 +3,30 @@
...
 
-- Lock screen to a PNG background displayed in multiple monitors, with enabled mouse pointer:
+- Lock the screen and don't hide the mouse pointer:
 
-`i3lock -i {{path/to/picture.png}} -p {{default|win}} -t`
+`i3lock --pointer {{default}}`
+
+- Lock the screen to a PNG background tiled over all monitors:
+
+`i3lock --image {{path/to/picture.png}} --tiling`
```
Comment (navarroaxel) [inline pages/linux/i3lock.md:28]:
```suggestion
`i3lock --image {{path/to/file.png}} --tiling`
```

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @marchersimon!

**Pull request merged**